app/models/region.rb

```diff
@@ -60,6 +61,21 @@ def log_payload
 
   private
 
+  def initialize_path?
+    @initialize_path
+  end
+
+  def initialize_path
+    errors.add(:reparent_to, "must assign new parent to initialize path") unless reparent_to
```

Comment:
This is unreachable code right? It will never be called if we have the `if: :reparent_to` upstairs.

Reply:
Yup, barring any sort of very very unlikely race condition. 👍